lib/game_team.rb

```diff
@@ -0,0 +1,36 @@
+class GameTeam
+
+attr_reader :game_id,
+            :team_id,
+            :hoa,
+            :result,
+            :settled_in,
+            :head_coach,
+            :goals,
+            :shots,
+            :tackles,
+            :pim,
+            :powerplay_opportunities,
+            :powerplay_goals,
+            :faceoff_win_percentage,
+            :giveaways,
+            :takeaways
+
+  def initialize(data)
+    @game_id = data[:game_id]
+    @team_id = data[:team_id]
+    @hoa = data[:hoa]
+    @result = data[:result]
+    @settled_in = data[:settled_in]
+    @head_coach = data[:head_coach]
+    @goals = data[:goals]
+    @shots = data[:shots]
+    @tackles = data[:tackles]
+    @pim = data[:pim]
+    @powerplay_opportunities = data[:powerplayopportunities]
+    @powerplay_goals = data[:powerplaygoals]
+    @faceoff_win_percentage = data[:faceoffwinpercentage]
```

Comment:
shouldn't the symbols on these lines have underscores in them?

Reply:
No! The hash that I'm referencing doesn't include underscores (it's how they gave us the data). I added the underscores in the variable names to adhere to ruby convention! Give it a shot in pry and you should see what I mean.